app/src/main/res/values/strings.xml

```diff
@@ -26,154 +26,156 @@
+    <string name="title_ssh_host_key_changed">The known host key for the SSH server changed!</string>
     <string name="title_continue">Continue</string>
-    <string name="title_delete_servers_before_import">Delete current Servers before Import</string>
+    <string name="title_delete_servers_before_import">Remove all prior servers before importing</string>
     <string name="title_password">Password</string>
-    <string name="title_use_repeater">Use Repeater</string>
-    <string name="title_private_key">Private Key</string>
+    <string name="title_use_repeater">Use repeater</string>
+    <string name="title_private_key">Private key</string>
     <string name="title_change">Change</string>
     <string name="title_about">About</string>
     <string name="title_license">Licence</string>
-    <string name="title_open_source_libraries">Open Source Libraries</string>
-    <string name="title_report_bug">Report a Bug</string>
-    <string name="title_github_repo">Github Repo</string>
+    <string name="title_open_source_libraries">Libre libraries</string>
+    <string name="title_report_bug">Report an issue</string>
+    <string name="title_github_repo">Source code</string>
 
-    <string name="hint_urlbar">Connect to a Server</string>
+    <string name="hint_urlbar">Server URL…</string>
     <string name="hint_server_name">Name (e.g. Work PC)</string>
     <string name="hint_host">Host</string>
-    <string name="hint_repeater_host">Repeater Host</string>
+    <string name="hint_repeater_host">Repeater host</string>
     <string name="hint_port">Port</string>
     <string name="hint_server_id">Server ID</string>
     <string name="hint_username">Username</string>
     <string name="hint_password">Password</string>
-    <string name="hint_key_password">Key Password</string>
+    <string name="hint_key_password">Key password</string>
 
-    <string name="msg_about_app">AVNC is an open-source VNC client for Android. It allows you to remotely control any device running a VNC server.</string>
-    <string name="msg_server_profile_added">Server added.</string>
-    <string name="msg_server_profile_deleted">Server deleted.</string>
-    <string name="msg_pip_not_supported">Picture-in-Picture is not supported on this device</string>
-    <string name="msg_ptr_hiding_not_supported">Hiding cursor is not supported on this device</string>
-    <string name="msg_loading">Loading …</string>
-    <string name="msg_imported">Imported Successfully</string>
-    <string name="msg_exported">Exported Successfully</string>
-    <string name="msg_export_auth_required">Authentication is required before Export.</string>
-    <string name="msg_export_auth_missing">Your device is not secured. Turn on screen lock to enable authentication before Export.</string>
-    <string name="msg_invalid_key_file">File is not a valid private key!</string>
+    <string name="msg_about_app">A copylefted libre VNC app.\nUse any device with a VNC server on it remotely.</string>
```

Comment:
```suggestion
    <string name="msg_about_app">A free and open-source VNC client.\nRemotely use any device running a VNC server.</string>
```

Reply:
Citing cost in the same sentence undersells the point of being libre.

Reply:
Really?

https://www.libreoffice.org/   -> "LibreOffice is Free and Open Source Software" (and this software has Libre in its name)
https://en.wikipedia.org/wiki/Linux_kernel -> "The Linux kernel is [a] free and open-source"
https://duckduckgo.com/?q=libre+site%3Awww.fsf.org&t=ffab&ia=web

Reply:
OK. Lets assume for a moment that I agree with you. So, here is your translation for `msg_about_app`:

```
Original source (English):       AVNC is an open-source VNC client for Android. It allows you to remotely control any device running a VNC server.

Translation (Norwegian Bokmål):  En fri VNC-klient.
                                 Koble til alle enheter som har en VNC-tjener.

Reverse translation (Google)  :  A free VNC client.
                                 Connect to any device that has a VNC server.
```

Lets ignore the fact that you changed the message (did you really think I won't notice it?), you yourself are describing it as *free* software. So please stop forcing me here. It's not happening.

And go ask normal users outside of your inner circle, have they even heard of 'libre' as in software freedom?

As you are a TDF member, you are in much better position to lead the change. Please start with the following pages, and 
stop wasting your effort here.

https://www.documentfoundation.org/
https://en.wikipedia.org/wiki/The_Document_Foundation
https://www.fsf.org/about/what-is-free-software

(See if you can find a single instance of the word 'libre' there)

Reply:
I am actually describing it as strictly ~libre software.
The reason I did so is to avoid the confusion with which a direct translation misses said point.
As this duality exists in English, there is no knowing what "free" means, other than the big "F".
Without even that, and not followed by "software" a point specifically about price is made,
unless the user is already in the know.
As suggested, and in my experience, both "Free" and "Open Source" which make for errors in translation.
"Open Source" in any other language is not trademarked, and thus does not carry the
implied meaning of all freedoms.
Since this is something the OSI went out of their way to avoid, it doesn't
carry much meaning, like intended. 23 years later and it doesn't stick.
People use "Open Source" every day, and they don't know.
If they do know, they often don't know the true meaning.

I am no closer to using "Open Source" as a reference than you, because
it is not a reference people have.
The important bit is to convey all freedoms, and in addition to these three,
AVNC is also copyleft. The word "libre" as the former ones, actually gives the user some chance
of not mistaking it for gratis, and also not ~1/4 freedoms and something that
has a reputation for ~"you can get this for free".
The reason I did so is it increases donations in the many projects that use it.
People do want to help something that can't be taken away,
and explaining that is IMO warranted.
I am not trying to slip anything past you here, which is evident from
me not closing the discussion, and my nb_NO translation.
At least 6 of these https://liberapay.com/explore/teams is where you find
copylefted libre or libre where it applies.
The point is otherwise to explain it, and both LibreOffice and the FSF
does that.

Reply:
If you want to explain freedom, please do. But don't assume everybody has to use your preferred words. And please don't assume someone is underselling these freedoms, even when their app is released on F-Droid before Play Store.